tests/lustre/typechecking/shouldPass/inlined_contract_03.lus

```diff
@@ -0,0 +1,9 @@
+
+node top(x: int) returns (y: int);
+(*@contract
+  var r: int = 5;
+  guarantee y >= (x + r);
```

Comment:
Ghost variable r should be visible in the guarantee.

Reply:
fixed.